.gitignore

```diff
@@ -25,12 +23,11 @@ bin/
 gen/
 out/
 
-# Gradle files
-.gradle/
-build/
+# Gradle build files
+matisse/build
+sample/build
+
```

Comment:
* .gradle is appeared before.
* `matisse/build` should be ignored not `/Matisse/build`